### PR TITLE
[haskell/en] some minor fixes

### PR DESCRIPTION
**Pull request by patrikja**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: haskell.html.markdown [hunk 1]
```diff
@@ -152,8 +152,8 @@ fib x
   | otherwise = fib (x - 1) + fib (x - 2)
 
 -- Pattern matching is similar. Here we have given three different
--- definitions for fib. Haskell will automatically call the first
--- function that matches the pattern of the value.
+-- equations for fib. Haskell will automatically use the first
```
Comment (vendethiel) [inline haskell.html.markdown]: The previous version was fine

File: haskell.html.markdown [hunk 2]
```diff
@@ -198,11 +198,11 @@ foo 5 -- 15
 -- multiplies the result of that by 4, and then returns the final value.
 foo = (4*) . (10+)
 
--- 4*(10 + 5) = 60
+-- 4*(10+ 5) = 60
```
Comment (vendethiel) [inline haskell.html.markdown]: Previois version was fine

File: haskell.html.markdown [hunk 3]
```diff
@@ -428,10 +427,8 @@ What is your name?
 Friend!
 Hello, Friend!
 
-```
-
```
Comment (vendethiel) [inline haskell.html.markdown]: That was correct

File: haskell.html.markdown [hunk 4]
```diff
@@ -428,10 +427,8 @@ What is your name?
...
 There's a lot more to Haskell, including typeclasses and monads. These are the
-big ideas that make Haskell such fun to code in. I'll leave you with one final
+big ideas that make Haskell so fun to code in. I'll leave you with one final
```
Comment (vendethiel) [inline haskell.html.markdown]: ...that's very nitpicky

File: haskell.html.markdown (hunk 1 above)
Comment (patrikja) [inline haskell.html.markdown]: The previous version was "fine" perhaps, but not correct. The three equations together constitute just one definition of the function fib. (See "4.4.3.1 Function bindings" of the Haskell 2010 report https://www.haskell.org/onlinereport/haskell2010/haskellch4.html#x10-620004 .)

File: haskell.html.markdown (hunk 2 above)
Comment (patrikja) [inline haskell.html.markdown]: Sure - either is fine, but my intention here was to help the reader match the sections exactly (to stress that "4*" and "10+" can be seen as unary functions).

File: haskell.html.markdown (hunk 3 above)
Comment (patrikja) [inline haskell.html.markdown]: I see - I missed that this closes the opening ```haskell very early in the file.

File: haskell.html.markdown (hunk 4 above)
Comment (patrikja) [inline haskell.html.markdown]: OK - now reverted.

**Comment by vendethiel** [comment]
Thanks.

**Pull request merged**